test/iron-dropdown-scroll-manager.html

```diff
@@ -99,6 +93,58 @@
           expect(Polymer.IronDropdownScrollManager.elementIsScrollLocked(parent))
             .to.be.equal(false);
         });
+
+        test('checking is disabled when no elements are locked', function() {
```

Comment:
Maybe rename this to "doesn't check locked elements when there are no locking elements" ?